Description: remove default optimizations from Makefile so we can use our own
Index: hardware-monitor-1.4.3/src/Makefile.am
===================================================================
--- hardware-monitor-1.4.3.orig/src/Makefile.am
+++ hardware-monitor-1.4.3/src/Makefile.am
@@ -28,7 +28,7 @@ hardware_monitor_SOURCES = \
                       value-history.cpp value-history.hpp \
                       ucompose.hpp
 
-AM_CXXFLAGS = $(DEPS_CFLAGS) $(SENSORS_CFLAGS) -Wall -O3
+AM_CXXFLAGS = $(DEPS_CFLAGS) $(SENSORS_CFLAGS)
 
 LDADD = $(DEPS_LIBS) $(SENSORS_LIBS)
 
